packages/cli/src/commands/generate/page/templates/page.js.template

```diff
@@ -1,14 +1,17 @@
-import { Link } from '@redwoodjs/router'
+import { Link, routes } from '@redwoodjs/router'
 
-const ${pascalName}Page = () => {
+const ${pascalName}Page = (<%= typeof param !== 'undefined' && param ? param.replace(/:[^}]+/, '') : '' %>) => {
```

Comment:
Is there any way you can move this logic outside of the template?

Reply:
This prompted me to do something I should have done from the start - write tests for the `handler` method and its `Listr` tasks.
I'm getting there, but not quite done yet. Will continue tomorrow

Reply:
Ooooh, that's some ambitious work!  Thanks Tobbe !

Reply:
Haven't fixed this yet, but I have the tests for `handler()` in place. Could you please take a look at how I test it? Is my approach sound? Also, if you could answer any of my questions I left as review comments on my own code that would be greatly appreciated 🙂

Reply:
I've moved the logic to a helper function in `page.js` that populates a bunch of template variables

Reply:
Amazing!